moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js

```diff
@@ -115,4 +115,7 @@ export const PROJECT_LIST_VIEW_EXPORT_CONFIG = {
   project_and_child_project_council_districts: {
     label: "Council districts",
   },
+  component_work_type_names: {
+    label: "Work types",
```

Comment:
@chiaberry Could you update the labels in this PR to "Component work types"? That is what I went with in the summary view because I almost added a tooltip but then I thought adding the "component" part to the name would be enough to point to where these values are coming from.

Open to other ideas here too!

Reply:
absolutely!